examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Machine Learning Workflows in T3C
+
+## Quickstart
+
+To run the T3C pipeline with a local front-end & server:
+1. Client-side: update `next-client/.env` to point to the report generation endpoint (e.g. `export PIPELINE_EXPRESS_URL=http://localhost:8080/generate`) and run `npm run dev`
+2. Server-side: update `express-pipeline/.env` with OpenAI/Anthropic/GCS keys and run `npm i && npm run dev`
+3. Navigate to `localhost:3000` and upload a CSV file from the UI. Make sure the CSV is formatted as follows 
+
+## Expected CSV format
+
+T3C expects a CSV file in a specific format:
+* there is ***no newline*** at the end of the file (which Pandas adds by default with `dataframe.to_csv()`)
+* the first line denoting the column format is exactly `id,comment`
```

Comment:
1. might be good to note that multimedia surveys will have additional columns? fine not to specify them in a quickstart though

2. it used to be comment-id and comment-body, did we change this? I like "id" and "comment" better tbh